src/controller/ExampleOperationalCredentialsIssuer.cpp

```diff
@@ -89,6 +89,49 @@ CHIP_ERROR ExampleOperationalCredentialsIssuer::Initialize(PersistentStorageDele
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ExampleOperationalCredentialsIssuer::GenerateNOCChainAfterValidation(NodeId nodeId, FabricId fabricId,
+                                                                                const Crypto::P256PublicKey & pubkey,
+                                                                                MutableByteSpan & rcac, MutableByteSpan & icac,
+                                                                                MutableByteSpan & noc)
+{
+    uint32_t nocLen = static_cast<uint32_t>(std::min(noc.size(), static_cast<size_t>(UINT32_MAX)));
```

Comment:
Can we just make `NewNodeOperationalX509Cert` take `size_t` for the `nocLen` bits so we don't need the extra casting to `uint32_t`?

Reply:
I updated the cert generation functions to use `MutableByteSpan`. That reduced a lot of boilerplate code.